src/comp/comp.py

```diff
@@ -24,48 +24,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [human.name for human in humans if human.name[0] == 'D']
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [human.name for human in humans if human.name[-1:] == 'e']
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [human.name for human in humans if human.name[0] == 'C' or human.name[0] == 'D' or human.name[0] == 'E' or human.name[0] == 'F' or human.name[0] == 'G']
```

Comment:
oh man this is a long one; try to see if you could simplify this one! :D

Try this: [Human.name for Human in humans if Human.name[0] in ['C', 'D', 'E', 'F', 'G']]